nmrfx-analyst-gui/src/main/resources/fxml/NMRControlRightSidePane.fxml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.StackPane?>
+
+
+<fx:root type="StackPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" />
```

Comment:
left as StackPane for now, but can update easily to something else here if needed Hbox, Vbox, AnchorPane etc, just not a plain Pane.